### ISSUE TITLE
Immunet flagged

### ISSUE THREAD

**Issue by GitNotus** [AcidWeb/CurseBreaker#338]

**Comment by AcidWeb** [comment]
Could you elaborate a little?

**Comment by GitNotus** [comment]
Yes, I am sorry . Antivirus Immunet (cisco AV) wont let me run CurseBreaker . Thank you for your work on this great addon,
Here is a partial log of the threat detected ":"Clam.Win.Malware.Pyspy-9952299-0","

**Comment by AcidWeb** [comment]
I can only suggest to use not broken AV.

File is clean: https://www.virustotal.com/gui/file/09669fa250d1a19434cd43ba4a24a6c433a59c1fdb5af04e24612a640608ae47
